GUIDE.md

```diff
@@ -0,0 +1,323 @@
+  for the running of the service through commercial/enterprise-level services
+  (that commercial customers pay for) and advertisements that are displayed on
+  the freely hosted documentation websites.  Some people think that it is
+  unethical to have advertisements for free and open source scientific
+  software websites (and particularly for NCAR, tax-funded work).  If this is a
+  concern you need to avoid, then I recommend hosting your documentation via
+  [GitHub Pages](https://pages.github.com/) which can let you host content,
+  such as documentation, directly from your GitHub repository.  To make this
+  work, however, we recommend that you create a CI job (e.g., via CircleCI)
+  to build _and commit_ your updated documentation for display on your GitHub
+  pages site ([see here](https://circleci.com/blog/deploying-documentation-to-github-pages-with-continuous-integration/)).
+
+CI services are the critical lynchpin of open source community software development,
+so they should be used whenever possible.
+
+I recommend setting your CI up to run _tests_ (including _linting_ tests)
+whenever a Pull Request is _created_ (and when edits/changes are made to an
+existing PR).  You can either use GitHub to block commits directly into the
+master branch (i.e., force all commits to go through PRs), or you can also
+run CI on changes made directly to your master branch.  I recommend setting
+up your CI to _build (and publish) documentation_ only when a PR is _merged_
+(i.e., only after it has been tested and approved).
+
+## 3. General Work Practices
+
+In general, most of what we do can be categorized as _Communication_ and/or
+_Development_.  However, there are a lot of aspects of what we do that fall
+outside of, or peripheral to, these practices.  As a team, this is the area
+of guidance that I believe we need to develop more.
+
+### 3.1 Periodically check your mirrors.
+
+Like driving a car, you need to periodically check your "mirrors" to see what
+is going on around you.  For us, that means checking your GitHub notifications
+periodically, checking your email, checking Zulip, _etc._.  However, you shouldn't
+go down the rabit hole whenever a new notification comes in.  Instead, you
+should assess whether the notification _does_ or _does not_ require a response.
+If the notification _does_ require a response, you must determine if that
+response can be delayed or must be immediate.  In general, I recommend short
+responses _immediately_ and longer responses (e.g., significant code modification)
+later, _after you are done with your current task_.  In fact, if a notification
+requires significant work, I would recommend an immediate _short_ response
+indicating that a _longer_ response will be coming.  I recommend this
+approach because it allows you to stay focused as much as possible.
+
+### 3.2 Be transparent.
+
+With everything that we do, we should strive to do it _in the open_.  That means
```

Comment:
:+1:

One thing that could be added here is the general use of formats like Markdown and CSV (which facilitate targeted discussions through line-by-line commenting) over proprietary or binary formats like Word or Excel when drafting documents.  

My https://github.com/NCAR/xdev/issues/60#issuecomment-564783755 was motivated by this.